src/cdk/constructs/cicd/index.test.ts

```diff
@@ -7,7 +7,7 @@ jest.mock('config')
 interface Context {
   [name: string]: string | any[] | any
 }
-describe.skip('CICD', () => {
+describe('CICD', () => {
   it('Creates a CICD stack with role as Caller Identity', async () => {
     const app = new App()
     app.node.tryGetContext = (): Context => ({
```

Comment:
tryGetConext is not used anymore in Mira, can you double-check if this stub is needed?

Reply:
That's right, it's not necessary.
Thanks 👍